opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/point.py

```diff
@@ -158,9 +182,34 @@ class ResourceMetrics:
     scope_metrics: Sequence[ScopeMetrics]
     schema_url: str
 
+    def to_json(self) -> str:
+        return dumps(
+            {
+                "resource": {
+                    "attributes": dict(self.resource.attributes.items()),
+                    "schema_url": self.resource.schema_url,
+                },
+                "scope_metrics": [
+                    loads(scope_metrics.to_json())
+                    for scope_metrics in self.scope_metrics
+                ],
+                "schema_url": self.schema_url,
+            }
+        )
+
 
 @dataclass(frozen=True)
 class MetricsData:
     """An array of ResourceMetrics"""
 
     resource_metrics: Sequence[ResourceMetrics]
+
+    def to_json(self) -> str:
```

Comment:
Nit: I think the typing for [formatter](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/export/__init__.py#L108) should be `opentelemetry.sdk.metrics.export.MetricsData` for the `Callable`. Also should probably rename `lamda metric:` to `lambda metrics_data`.

Reply:
Fixed